test/checks/keyboard/no-focusable-content.js

```diff
@@ -28,17 +28,22 @@ describe('no-focusable-content tests', function() {
     assert.isTrue(noFocusableContent(null, null, vNode));
   });
 
-  it('should return false if element has focusable content', function() {
+  it('should return true if element has content which is focusable (tabindex=0) and does not have a widget role', function() {
     var params = checkSetup(
       '<button id="target"><span tabindex="0">Hello</span></button>'
     );
 
-    assert.isFalse(noFocusableContent.apply(checkContext, params));
-    assert.deepEqual(checkContext._data, null);
-    assert.deepEqual(checkContext._relatedNodes, [params[2].children[0]]);
+    assert.isTrue(noFocusableContent.apply(checkContext, params));
   });
 
-  it('should return false if element has natively focusable content', function() {
+  it('should return true if element has content which has negative tabindex and non-widget role', function() {
```

Comment:
Can you add an example that uses span, tabindex and some widget role that shows those failing?